frontend/src/metabase-lib/lib/Question.ts

```diff
@@ -925,7 +927,7 @@ export default class Question {
     clean?: boolean;
     query?: Record<string, any>;
     includeDisplayIsLocked?: boolean;
-    creationType: string;
+    creationType?: string;
```

Comment:
`creationType` here is used by Question's `getUrl` method and it can actually handle an undefined `creationType`, so I've marked it as an optional here